src/client/visualizations/line-chart/charts/charts-per-split/label.tsx

```diff
@@ -29,8 +29,9 @@ interface LabelProps {
 export const Label: React.SFC<LabelProps> = props => {
   const { essence, datum } = props;
   if (hasNominalSplit(essence)) {
+    const nominalSplit = getNominalSplit(essence);
     const nominalDimension = getNominalDimension(essence);
-    const splitValue = datum[nominalDimension.name] as PlywoodValue;
+    const splitValue = nominalSplit.selectValue(datum);
     return <div className="split-chart-label">
     <span className="split-chart-dimension-title">
       {nominalDimension.title}
```

Comment:
Can we get title from split or do we need to get it from dimension?

Reply:
`title` is defined on Dimension (by user via configuration file). Dimensions are basically read-only but complex so we don't want to put them on splits which are often serialised and deserialised.